### PR TITLE
Documentation and sample edge shields for Worker Insights.

### PR DESCRIPTION
**Pull request by fxulusoy**

### CODE REVIEW

File: worker-insights/README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# Edge Shields for IBM IoT Worker Insights
+
```
Comment (dragos-cojocari) [inline worker-insights/README.md]: We should add the link to the KC here: https://www.ibm.com/support/knowledgecenter/SSQNYQ_bas/iot-insurance/safer_workplace/worker_summary.html

File: worker-insights/README.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+# Prerequisites
+1. Download edge-processing-toolkit from Worker Insights API with your account.
```
Comment (dragos-cojocari) [inline worker-insights/README.md]: Let's provide a link to either this topic https://www.ibm.com/support/knowledgecenter/SSQNYQ_bas/iot-insurance/iotinsurance_rest_apis.html?pos=2 or this one https://www.ibm.com/support/knowledgecenter/SSQNYQ_bas/iot-insurance/iotinsurance_shield_toolkit.html The latter should be better but at this time it's all about home insights

File: worker-insights/README.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+production builds, you should use `npm run package`. Uglification and minification
+are applied for the production builds. Detailed steps are below:
+
+* Add your shield code to module.export[0].entry in the `<sw-edge-shields>/webpack.config.js`
+* Run `npm run build` or `npm run package`
+* You will find the generated packed file in `<sw-edge-shields>/dist/` directory
+* Create a shield and upload the shield code file via the dashboard
+
+## Test
+Tests are located under tests directory. As long as we don't need webpack pre
```
Comment (dragos-cojocari) [inline worker-insights/README.md]: "As long as we don't need webpack " - is that sentence needed given that above we explicitly state we use webpack?

Comment (fxulusoy) [reply worker-insights/README.md]: Yes. We want to emphasize that we don't need webpack to run tests.

**Pull request merged**